### PR TITLE
Initialized dashboard with sidebar

### PR DESCRIPTION
**Pull request by ken1000minus7**
- Edited the new dashboard layout with sidebar
- Made the sidebar responsiveness
- Moved the social links on the footer to a common component to be reused
- Began work on home page of dashboard

### CODE REVIEW

File: components/dashboard/Sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <>
+            <AiOutlineMenuUnfold
+                onClick={() => setMobileSidebarOpen("true")}
+                className={styles.hamburger}
+            />
+            <div className={styles.sidebar} data-open={mobileSidebarOpen}>
+                <div className={styles.header}>
+                    <div className={styles.pfp}>
+                        {/* eslint-disable-next-line @next/next/no-img-element */}
+                        <img src={user.dp} alt={""} />
```
Comment (Ishwarendra) [inline components/dashboard/Sidebar.tsx:59]: Use `<Image>` from `next/image`. We will change it in older pages too.

File: styles/components/SidebarItem.module.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
+@import "../variables";
+
+.sidebarItem {
+    display: flex;
+    flex-direction: row;
+    width: 90%;
+    padding: 5px;
+    border-radius: 10px;
+    font-size: 25px;
+    transition: 400ms;
```
Comment (Ishwarendra) [inline styles/components/SidebarItem.module.scss:10]:
```suggestion
```

If this class is related to hover effect on sidebar items then remove it. Incase it is not, find it and remove that transition property.

image

File: components/dashboard/Sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        name: "Ken",
+        designation: "Member",
+        dp: "https://avatars.githubusercontent.com/u/78747188?v=4"
+    }
+
+    const [mobileSidebarOpen, setMobileSidebarOpen] = useState("false")
+
+    return (
+        <>
+            <AiOutlineMenuUnfold
```
Comment (Ishwarendra) [inline components/dashboard/Sidebar.tsx:51]: Need to change icon too, but we will decide this later.

**Review by Ishwarendra — CHANGES_REQUESTED** [review]
Need

File: styles/components/SidebarItem.module.scss (hunk 2 above)
Comment (ken1000minus7) [reply styles/components/SidebarItem.module.scss:10]: Why should we remove this though? 🤔

File: components/dashboard/Sidebar.tsx (hunk 3 above)
Comment (ken1000minus7) [reply components/dashboard/Sidebar.tsx:51]: Will replace with hamburger component of navbar

**Comment by Ishwarendra** [comment]
Can we keep hovered item and active item of sidebar styles different?

**Comment by ken1000minus7** [comment]
@Ishwarendra Yeah can do, what color should we use for active? 🤔

**Review by harshjohar — APPROVED** [review]
LGTM!

**Pull request merged**